server/src/com/sixsq/slipstream/ssclj/resources/spec/device.cljc

```diff
@@ -38,6 +38,9 @@
 (s/def :cimi.device/powerPlugged boolean?)
 (s/def :cimi.device/agentType ::cimi-core/nonblank-string)
 (s/def :cimi.device/actuatorInfo ::cimi-core/nonblank-string)
+(s/def :cimi.device/sensorType ::cimi-core/nonblank-string)
```

Comment:
Should this still be in `device` now that it's in `device-dynamic`?

Reply:
No, I have already made the changes and created a new PR....just waiting for the @cjdcordeiro to merge the PR.....I think he will do it, once I'll update my code for the resource-categorization...